packages/firebase_remote_config/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
 dev_dependencies:
```

Comment:
I think you forgot to update the pubspec version.

Reply:
done, thanks!